source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
+|         |            |     |       |          |          | run on the C5    |
+|         |            |     |       |          |          | login nodes.     |
++         +------------+-----+-------+----------+----------+------------------+
+|         | eslogin_c6 | 1   | *max* | 12:00:00 | 16:00:00 | These jobs will  |
+|         |            |     |       |          |          | run on the C5    |
+|         |            |     |       |          |          | login nodes.     |
++         +------------+-----+-------+----------+----------+------------------+
+|         | dtn_f5_f6  | 513 | *max* | 12:00:00 | 16:00:00 | These jobs will  |
+|         |            |     |       |          |          | run on the DTN   |
+|         |            |     |       |          |          | nodes.  The DTN  |
+|         |            |     |       |          |          | nodes have both  |
+|         |            |     |       |          |          | F5 and F6        |
+|         |            |     |       |          |          | mounted.         |
++         +------------+-----+-------+----------+----------+------------------+
+|         | cron_c5    | 1   | *max* | 12:00:00 | 16:00:00 | Required         |
+|         |            |     |       |          |          | partition for    |
+|         |            |     |       |          |          | jobs run under   |
+|         |            |     |       |          |          | scron on the C5  |
+|         |            |     |       |          |          | login nodes.     |
++         +------------+-----+-------+----------+----------+------------------+
+|         | cron_c6    | 1   | *max* | 12:00:00 | 16:00:00 | Required         |
+|         |            |     |       |          |          | partition for    |
+|         |            |     |       |          |          | jobs run under   |
+|         |            |     |       |          |          | scron on the C6  |
+|         |            |     |       |          |          | login nodes.     |
++---------+------------+-----+-------+----------+----------+------------------+
 
-**module Switch [available_module] replacement_module:** Switch loaded
-modulefile1 with modulefile2. If modulefile1 is not specified, then it
-is assumed to be the currently loaded module with the same root name
-as modulefile2
+.. note::
 
-**module swap [available_module] replacement_module:** Switch loaded
-modulefile1 with modulefile2. If modulefile1 is not specified, then it
-is assumed to be the currently loaded module with the same root name
-as modulefile2
+    The partition information above, and additional information can be listed
+    using the :command:`scontrol --cluster <cluster> show parition` where
+    :command:`<cluster>` is the name of one of the available clusters.
+
+Job Dependencies
+----------------
+
+Oftentimes, a job will need data from some other job in the queue, but it's
```

Comment:
Recommend Frequently rather than oftentimes.

Reply:
Changed in a new push.